src/app/sharedUI/screens/Artwork/ArtworkContent/ArtworkContent.tsx

```diff
@@ -75,7 +75,7 @@ export const ArtworkContent = ({ slug }: { slug: string }) => {
   // For Presentation Mode
   const isPriceHidden = GlobalStore.useAppState((state) => state.presentationMode.hiddenItems.price)
```

Comment:
```suggestion
  const showPrice = !GlobalStore.useAppState((state) => state.presentationMode.hiddenItems.price)
```
I wonder if that makes it clearer